modules/pvc-autoscale/data/pvc_autoscale_notebook.json

```diff
@@ -0,0 +1,162 @@
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | app =~ \"shoreline\" | limit=1 | `cat  \"/tmp/pvc_test.yaml\"`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "Verify that pvc-test pods have been created.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | app = \"pvc-test\"",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "## Install Packages in Pods",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "host | pod | app = \"pvc-test\" | `apt-get update`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "host | pod | app = \"pvc-test\" | `apt-get install -y jq iproute2 python3 procps psmisc`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "host | pod | app = \"pvc-test\" | `curl -LO https://dl.k8s.io/release/v1.20.0/bin/linux/amd64/kubectl`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "host | pod | app = \"pvc-test\" | `chmod +x kubectl; mv kubectl /bin/`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "# PVC Autoscale",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "## Check used percentage of disk\nChecking the usage of '/tofill' disk. The exit code is the percentage of the disk",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | name =~ \"pvc-test\" | testpvc_check_disk(\"tofill\")",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "## List pods with used percentage of disk greater than a threshold\nPods with used percentage of disk '/tofill' greater than 30 percent",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | name =~ \"pvc-test\" | filter( testpvc_check_disk('tofill') >= 30 )",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "## Fill disk\nFill disk with file of size 300MB.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pod | app='pvc-test' | `head -c 300000000 /dev/zero > /tofill/bar1`",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "OP_LANG"
+    },
+    {
+      "content": "Check disk usage after filling the disk.",
+      "enabled": false,
+      "name": "unnamed",
+      "type": "MARKDOWN"
+    },
+    {
+      "content": "pods | name =~ \"pvc-test\" | testpvc_check_disk(\"tofill\")",
```

Comment:
Even though I only paused a couple seconds, the alarm had already triggered I ran this command and the filter below. So the results looked "wrong".
Not sure what the best change would be, but maybe bundle this command with the one above, and add a note to the filter command below to check if the alarm has already triggered.